packages/component-library/src/components/plmg-tabs/plmg-tabs.stories.js

```diff
@@ -0,0 +1,94 @@
+export default {
+  title: 'Component/Tabs',
+  parameters: {},
+  decorators: [],
+  argTypes: {},
+};
+
+export const Tabs = () => {
+  const htmlContent = `
+  <plmg-tabs>
+    <plmg-tab active label="Tab 1"></plmg-tab>
```

Comment:
Yes, when implementing we need to set the initial active state (if it exists, we can have a tabs list that isn't active). After initially being set handling of active state is taken by the component.
Is that confusing? Maybe if the active prop was on `<plmg-tabs>` with the tab id as the argument rather than on the specific tab element.